src/VirtualClient/VirtualClient.Main/profiles/PERF-POSTGRESQL-SYSBENCH-OLTP.json

```diff
@@ -182,7 +182,7 @@
             "Parameters": {
                 "Scenario": "DownloadSysbenchPackage",
                 "BlobContainer": "packages",
-                "BlobName": "sysbench-1.0.20.rev2.zip",
+                "BlobName": "sysbench-1.0.20.rev3.zip",
```

Comment:
Is the clean up python the only change in rev3? Please put the scripts as scripts in VC source, in that way I could review the changes. Otherwise you need to document what has changed that are not in the source